mlflow/telemetry/events.py

```diff
@@ -105,3 +105,11 @@ def parse(cls, arguments: dict[str, Any]) -> dict[str, Any] | None:
 
 class PromptOptimizationEvent(Event):
     name: str = "prompt_optimization"
+
+
+class LogMetricEvent(Event):
+    name: str = "log_metric"
+
+    @classmethod
+    def parse(cls, arguments: dict[str, Any]) -> dict[str, Any] | None:
+        return {"synchronous": arguments.get("synchronous")}
```

Comment:
```suggestion
        return {"synchronous": arguments.get("synchronous", True)}
```

do we need default value?

Reply:
We don't, arguments bound default value already.